skeletons/web-extension/content-script.js

```diff
@@ -90,7 +90,7 @@
       if (!injected) {
         // cannot use eval here, as the context is limited to the content script-
         const elem = document.createElement('script') ;
-        elem.textContent = message.value;
+        elem.src =  chrome.runtime.getURL('panes-3-16-0/ember_debug.js');
```

Comment:
Will specifically loading 3-16-0 here break our check where we switch to different versions for older Ember?

Reply:
Yes, just testing now. I will change it later